### ISSUE TITLE
A11y_.NETOnlineREPL_.NETEditor_Screenreader: NVDA announces unnecessary hidden content after announcing code data while navigating using down arrow key.

### ISSUE THREAD

**Issue by pranshu512** [dotnet/try#907]
[“Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_.NET Online REPL_Web_Feb2021;#.NET Online REPL;#Win10;#FTP;#DesktopWeb;#WCAG1.3.1;#Screenreader;#Narrator;#NVDA;#Chrome;#ChromiumEdge;#Element:Text;

### **Actual:**
NVDA announces unnecessary hidden content after announcing code data while navigating using down arrow key.

**Same issue is also observed with Chromium Edge +Narrator.**

### **Expected:**
NVDA should not announce unnecessary hidden content after announcing code data while navigating using down arrow key.

### **Environment Details:**
Application Name: .NET Online REPL
Operating System: Windows10
Browser Version : Google Chrome Version 87.0.4280.141 (Official Build) (64-bit)
Chromium Edge Version 89.0.774.14 (Official build) dev (64-bit)
Screenreader: NVDA 2020.3,Narrator

### **Repro Steps:**

1. Open .NET Editor coding screen .
2. Start NVDA and navigate to .NET editor section with the help of down arrow key.
3. Check whether NVDA announces unnecessary hidden content after announcing code data.

### **User Impact:**
Screen reader user will get incorrect unnecessary information and get confused.

**Comment by fsteffi** [comment]
env:https://learn.microsoft.com/en-us/dotnet/csharp/tour-of-csharp/tutorials/hello-world?tutorial-step=1&WT.mc_id=-blog-scottha
[Screen Recording (81).webm](https://user-images.githubusercontent.com/24580570/203726896-2bea21f4-52f9-4e5d-a126-16456278382b.webm)

**Comment by AnKushSingh05** [comment]
#Closed;
Verified the issue in below environment. Issue is fixed hence closing the bug.
https://learn.microsoft.com/en-us/dotnet/csharp/tour-of-csharp/tutorials/hello-world?tutorial-step=1&WT.mc_id=-blog-scottha

https://github.com/user-attachments/assets/184a13bc-5130-4394-84c7-0d1b8aec9b83
